host-configs/docker/clang@10.0.0.cmake

```diff
@@ -35,8 +35,6 @@ set(CMAKE_C_FLAGS "-fPIC -pthread" CACHE STRING "")
 
 set(CMAKE_CXX_FLAGS "-fPIC -pthread" CACHE STRING "")
 
-set(CMAKE_GENERATOR "Unix Makefiles" CACHE STRING "")
```

Comment:
@samuelpmish This is now gone.

Reply:
how did this go away? spack just got rid of it?